### ISSUE TITLE
Issue destroying web apps popped up in the last week or so

### ISSUE THREAD

**Issue by ZacharyWallace** [Broadcom/terraform-provider-luminate#23]
I'm receiving the following errors when deleting web apps. This is a recent development. The web apps are being deleted, but it's like the state isn't saving because the provider is causing some sort of crash. Terraform is v 0.14.9. Still not exactly sure what's going on here, if there is anything further I can provide please let me know.

```
Do you really want to destroy all resources?
  Terraform will destroy all your managed infrastructure, as shown above.
  There is no undo. Only 'yes' will be accepted to confirm.

  Enter a value: yes

luminate_web_application.sh1-api-app: Destroying... [id=bd78a7f6-a98c-45f3-884b-c7d282776b10]
luminate_web_application.hf1-web-app: Destroying... [id=125db742-d5a9-458f-be9f-a5a096a03a0f]
luminate_web_application.lic-web-app: Destroying... [id=43fb2c48-fe8c-4984-9939-556523fda0a1]
luminate_web_application.sh1-web-app: Destroying... [id=6c1a24a5-d737-4743-911c-04014a17dd60]

Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing


panic: runtime error: invalid memory address or nil pointer dereference
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7b7b1c]
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9:
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: goroutine 26 [running]:
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/Broadcom/terraform-provider-luminate/service/dto.ConvertToApplicationDTO(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/service/dto/application_converter.go:16 +0x5c
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/Broadcom/terraform-provider-luminate/service.(*ApplicationAPI).GetApplicationById(0xc000010120, 0x0, 0x0, 0x2, 0xc00055a3c8, 0xc00055a3c8)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/service/application.go:86 +0x12b
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/Broadcom/terraform-provider-luminate/provider.resourceReadWebApplication(0xc00010c3f0, 0xecd680, 0xc0002ec180, 0x0, 0x0)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/provider/resource_web_application.go:170 +0xc2
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/Broadcom/terraform-provider-luminate/provider.resourceDeleteWebApplication(0xc00010c3f0, 0xecd680, 0xc0002ec180, 0x24, 0x1a94e00)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/provider/resource_web_application.go:226 +0x11d
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc000109e00, 0xc0000ba4b0, 0xc000215be0, 0xecd680, 0xc0002ec180, 0x40e201, 0x20, 0x1050940)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:259 +0x498
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc0004a8a00, 0xc0004cd908, 0xc0000ba4b0, 0xc000215be0, 0x1a94e00, 0xc0000ba4b0, 0x0)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:285 +0x99
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/hashicorp/terraform/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc000010bf0, 0x13211e0, 0xc0004e01e0, 0xc0003b2480, 0xc000010bf0, 0xc0004e01e0, 0xc00029aa80)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/github.com/hashicorp/terraform/helper/plugin/grpc_provider.go:851 +0x892
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: github.com/hashicorp/terraform/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1091520, 0xc000010bf0, 0x13211e0, 0xc0004e01e0, 0xc0000ba230, 0x0, 0x13211e0, 0xc0004e01e0, 0xc0004e6380, 0x310)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/github.com/hashicorp/terraform/internal/tfplugin5/tfplugin5.pb.go:3217 +0x217
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: google.golang.org/grpc.(*Server).processUnaryRPC(0xc0004b6480, 0x132ce80, 0xc0004b7380, 0xc0004da100, 0xc0000a33b0, 0x1a63500, 0x0, 0x0, 0x0)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/google.golang.org/grpc/server.go:966 +0x46a
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: google.golang.org/grpc.(*Server).handleStream(0xc0004b6480, 0x132ce80, 0xc0004b7380, 0xc0004da100, 0x0)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/google.golang.org/grpc/server.go:1245 +0xd16
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc00013e210, 0xc0004b6480, 0x132ce80, 0xc0004b7380, 0xc0004da100)
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/google.golang.org/grpc/server.go:685 +0xbb
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: created by google.golang.org/grpc.(*Server).serveStreams.func1
2022-12-13T20:13:07.012Z [DEBUG] plugin.terraform-provider-luminate_v1.0.9: 	/home/circleci/project/vendor/google.golang.org/grpc/server.go:683 +0xa1
2022-12-13T20:13:07.013Z [DEBUG] plugin: plugin process exited: path=.terraform/providers/terraform.company.com/broadcom/luminate/1.0.9/linux_amd64/terraform-provider-luminate_v1.0.9 pid=9008 error="exit status 2"
2022/12/13 20:13:07 [DEBUG] luminate_web_application.hf1-web-app: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2022/12/13 20:13:07 [TRACE] EvalWriteState: writing current state object for luminate_web_application.hf1-web-app
2022/12/13 20:13:07 [DEBUG] luminate_web_application.sh1-web-app: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2022/12/13 20:13:07 [DEBUG] luminate_web_application.lic-web-app: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2022/12/13 20:13:07 [TRACE] EvalWriteState: writing current state object for luminate_web_application.lic-web-app
2022/12/13 20:13:07 [DEBUG] luminate_web_application.sh1-api-app: apply errored, but we're indicating that via the Error pointer rather than returning it: rpc error: code = Unavailable desc = transport is closing
2022/12/13 20:13:07 [TRACE] EvalWriteState: writing current state object for luminate_web_application.sh1-api-app
2022/12/13 20:13:07 [TRACE] EvalWriteState: writing current state object for luminate_web_application.sh1-web-app
2022/12/13 20:13:07 [TRACE] vertex "luminate_web_application.hf1-web-app (destroy)": visit complete
2022/12/13 20:13:07 [TRACE] vertex "luminate_web_application.lic-web-app (destroy)": visit complete
2022/12/13 20:13:07 [TRACE] vertex "luminate_web_application.sh1-api-app (destroy)": visit complete
2022/12/13 20:13:07 [TRACE] vertex "luminate_web_application.sh1-web-app (destroy)": visit complete
2022/12/13 20:13:07 [TRACE] dag/walk: upstream of "provider[\"terraform.company.com/broadcom/luminate\"] (close)" errored, so skipping
2022/12/13 20:13:07 [TRACE] dag/walk: upstream of "meta.count-boundary (EachMode fixup)" errored, so skipping
2022/12/13 20:13:07 [TRACE] dag/walk: upstream of "root" errored, so skipping
2022-12-13T20:13:07.015Z [DEBUG] plugin: plugin exited
```

**Comment by satellite-no** [comment]
Seeing same issue!

**Comment by ZacharyWallace** [comment]
This issue is resolved with the new version of the provider, 1.1.1.
